sovrin_client/agent/endpoint.py

```diff
@@ -29,7 +29,8 @@ def __init__(self, port: int, msgHandler: Callable,
         if basedirpath:
             stackParams["basedirpath"] = basedirpath
 
-        super().__init__(stackParams, self.baseMsgHandler)
+        self.registry = {}
+        super().__init__(stackParams, self.baseMsgHandler, self.registry)
```

Comment:
The two lines above can be done with:
+        super().__init__(stackParams, self.baseMsgHandler, {})

The super init assigns self.registry.